site/content/docs/main/data-movement-node-selection.md

```diff
@@ -85,9 +87,9 @@ Here is a sample of the ConfigMap with ```loadAffinity```:
 }
 ```
 
-This example demonstrates how to use both `matchLabels` and `matchExpressions` in the same single LoadAffinity element.
+This example demonstrates how to use both `matchLabels` and `matchExpressions` in the same single LoadAffinity element. This is the recommended approach since only the first element in the array is processed.
```

Comment:
I don't think we need this one, for each of the example, as we have already clarified it as an overall statement.